feature/menu/src/main/java/com/guidal/feature/menu/privacy/PrivacyScreen.kt

```diff
@@ -0,0 +1,184 @@
+                    Switch(
+                        checked = uiState.isLocationEnabled,
+                        onCheckedChange = { checked ->
+                            if (checked) {
+                                requestLocationPermissionLauncher.launch(
+                                    arrayOf(
+                                        Manifest.permission.ACCESS_FINE_LOCATION,
+                                        Manifest.permission.ACCESS_COARSE_LOCATION
+                                    )
+                                )
+                            } else {
+                                privacyViewModel.openAppSettings()
+                            }
+                        }
+                    )
+                }
+
+                Text(
+                    text = stringResource(R.string.privacy_location_permission_description),
+                    style = MaterialTheme.typography.bodyMedium,
+                    color = MaterialTheme.colorScheme.onSurfaceVariant,
+                    textAlign = TextAlign.Justify,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                )
+
+                HorizontalDivider(
+                    modifier = Modifier.padding(top = 8.dp)
+                )
+            }
+
+            // Collection Permission
+            Column(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(horizontal = 20.dp)
+            ) {
+                Row(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .height(54.dp),
+                    verticalAlignment = Alignment.CenterVertically
+                ) {
+                    Text(
+                        text = stringResource(R.string.privacy_collection_permission_label),
+                        style = MaterialTheme.typography.bodyLarge,
+                        modifier = Modifier
+                            .weight(1f)
+                    )
+                    Switch(
+                        checked = uiState.isCollectionEnabled,
+                        // TODO Implement collection
+                        onCheckedChange = { checked ->
+                            privacyViewModel.updateCollectionState(
+                                checked
+                            )
+                      },
+                    )
+                }
+
+                Text(
+                    text = stringResource(R.string.privacy_collection_permission_description),
+                    style = MaterialTheme.typography.bodyMedium,
+                    color = MaterialTheme.colorScheme.onSurfaceVariant,
+                    textAlign = TextAlign.Justify,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                )
+
+                HorizontalDivider(
+                    modifier = Modifier.padding(top = 8.dp)
+                )
```

Comment:
Repeating code, could look into mapping buttons to a component, but currently this implementation works too